executor/sort.go

```diff
@@ -59,6 +59,8 @@ type SortExec struct {
 	multiWayMerge *multiWayMerge
 	// spillAction save the Action for spill disk.
 	spillAction *chunk.SortAndSpillDiskAction
+
+	childrenUsed []int
```

Comment:
What about `columIdxsUsedByChild` ? `childrenUsed ` is hard to understand. Also, please add the comment to explain its usage.